src/main/java/io/rpg/view/LocationView.java

```diff
@@ -27,7 +35,7 @@ public class LocationView extends Scene
 
   private final Set<KeyboardEvent.Observer> onKeyPressedObservers;
 
-  private final LocationViewModel viewModel;
+  private static LocationViewModel viewModel;
```

Comment:
Dlaczego `static`? Wydaje mi się, że niepotrzebnie.

Reply:
wlasnie tez nad tym myslalem
dalem static, bo uzywam tego w fromConfig(), ktore jest statyczne
ale nie jestem pewny czy lepiej zrobic to pole statyczne czy np je przekazac do tej metody jako parametr